.azure/azure-pipelines.ci.yml

```diff
@@ -224,6 +224,15 @@ stages:
       tls: stub
       extraBuildArgs: -SanitizeAddress
       config: Debug
+  - template: ./templates/build-config-user.yml
```

Comment:
This should be moved down into the "Build Linux - Non Tested" stage so that the rest of the pipeline (perf & tests) is not delayed by this job.